pygamelib/engine.py

```diff
@@ -2697,6 +2698,8 @@ def _ref2obj(ref):
                 local_object.name = ref["name"]
             if "model" in obj_keys:
                 local_object.model = ref["model"]
+            if "sprixel" in obj_keys:
+                local_object.sprixel = core.Sprixel.from_ansi(ref["sprixel"])
```

Comment:
If you use `Sprixel.serialize()` you need need to use `Sprixel.load()` here (https://pygamelib.readthedocs.io/en/latest/gfx_core.html#pygamelib.gfx.core.Sprixel.load).
From ansi does not load a serialized sprixel (it loads an ANSI sequence like the one generated by the climage python module).